sdk/communication/azure-communication-callingserver/README.md

```diff
@@ -0,0 +1,215 @@
+[//]: # ({x-version-update-end})
+
+## Key concepts
+
+At a high level the Azure Communication CallingServer API will support 2 kinds of scenarios:
+
+- In-call app: Contoso server app is a participant in the call.  
+
+- Out-call app: Contoso server app is not a participant in the call - Server app can subscribe to events for calls between specific users or even all users belonging to the ACS azure resource.  
+
+Based on if the Contoso app join a call or not, APIs can be divided into two categories:   
+
+- In-call APIs: Contoso app is one of the participant in a call. It can be applicable for app to person (A2P) or person to app (P2A) case, or multi-party/group calls that server apps joined as a participant to provide audio/prompt.  
+
+- Out-of-call APIs: Contoso app can invoke these set of APIs without joining a call. It is applicable for actions on P2P calls, A2P calls, P2A calls and group calls.  
+
+## Examples
+
+### Create CallClient
+
+Once you initialized a `CallClient` class, you can do the following chat operations:
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L31-L40 -->
+```java
+// You can find your endpoint and access key from your resource in the Azure Portal
+String endpoint = "https://<RESOURCE_NAME>.communcationservices.azure.com";
+
+// Your connectionString retrieved from your Azure Communication Service
+String connectionString = "https://<resource-name>.communication.azure.com/;<access-key>";
+
+// Initialize the call client
+final CallClientBuilder builder = new CallClientBuilder();
+builder.endpoint(endpoint)
+    .connectionString(connectionString);
+CallClient callClient = builder.buildClient();
+```
+
+### Create, AddParticipant, Hangup and Delete a call
+
+Create a Call: 
+<!-- embedme src/samples/java/com/azure/communication/callingserver/ReadmeSamples.java#L52-L79 -->
+```java
+CommunicationIdentifier source = new CommunicationUserIdentifier("<acs-user-identity>");
+
+List<CommunicationIdentifier> targets = new ArrayList<>();
+
+CommunicationIdentifier firstCallee = new CommunicationUserIdentifier("<acs-user-identity-1>");
+CommunicationIdentifier secondCallee = new CommunicationUserIdentifier("<acs-user-identity-2>");
+
+targets.add(firstCallee);
+targets.add(secondCallee);
+
+String callbackUri = "<callback-uri-for-notification>";
+
+List<CallModality> requestedModalities = new ArrayList<>();
+requestedModalities.add(CallModality.AUDIO);
+requestedModalities.add(CallModality.VIDEO);
+
+List<EventSubscriptionType> requestedCallEvents = new ArrayList<>();
+requestedCallEvents.add(EventSubscriptionType.DTMF_RECEIVED);
+requestedCallEvents.add(EventSubscriptionType.PARTICIPANTS_UPDATED);
+
+CreateCallOptions createCallOptions = new CreateCallOptions(
+    callbackUri,
+    requestedModalities,
+    requestedCallEvents);
+
+CreateCallResponse createCallResult =  callClient.createCall(source, targets, createCallOptions);
+
+String callId = createCallResult.getCallLegId();
```

Comment:
I dont think we need this line in the samples

Reply:
line removed